common/lib/client/resource.js

```diff
@@ -1,5 +1,8 @@
 var Resource = (function() {
 	var msgpack = (typeof(window) == 'object') ? window.Ably.msgpack : require('msgpack-js');
+	var isErrATokenProblem = function(error) {
```

Comment:
Looks like this was merged into `master`?  I have corrected in https://github.com/ably/ably-js/commit/07e6e3b9ca88f34f6a1ac5e6c657ce068f16788b, are you happy with that?